app/services/api_tracking_information.rb

```diff
@@ -0,0 +1,39 @@
+class ApiTrackingInformation
+  require 'open-uri'
+  require 'pry'
+
+  def self.tracking_by(**args)
+    begin
+      if args[:id]
+        api_url = "https://api.aftership.com/v4/trackings/#{args[:id]}"
+      elsif args[:slug] && args[:tracking_number]
+        api_url = "https://api.aftership.com/v4/trackings/#{args[:slug]}/#{args[:tracking_number]}"
+      end
+
+      serialized = open(
+        api_url,
+        "aftership-api-key" => "711623ab-a860-4d3a-84d1-bfa891ff5058",
```

Comment:
Normally is a password/credentials we used for access private resources, you should either put into https://www.engineyard.com/blog/encrypted-rails-secrets-on-rails-5.1, or make it hidden somewhere without commiting to Git!